vault/yaml/osfamilymap.yaml

```diff
@@ -0,0 +1,8 @@
+# -*- coding: utf-8 -*-
```

Comment:
Likewise, as the [comment above](https://github.com/saltstack-formulas/vault-formula/pull/23/files?file-filters%5B%5D=.sls&file-filters%5B%5D=.yaml&hide-deleted-files=true#r268416081).